Makefile

```diff
@@ -56,11 +56,7 @@ PKGCONFIG_DIR=$(PREFIX)/share/pkgconfig/
 
 CARGO_FLAGS+=$(VARIANT_FLAGS)
 
-ifneq ($(filter Darwin %BSD,$(shell uname -s)),)
-    INSTALL?=install
-else
-    INSTALL?=ginstall
-endif
+INSTALL?=install
```

Comment:
shouldn't that statement not come *before* the infneq?

Reply:
> shouldn't that statement not come _before_ the infneq?

There is no `ifneq` anymore, `install` is used per default, unless overriden by the caller.

I personally would prefer to keep the upstream logic (now that I understand a little more about it), that is, using `install` on linux, `ginstall` on macOS. If that `ginstall` is problematic for homebrew users, then my 2nd preference is just use `install`, also on macOS.

Reply:
I see